website/docs/usage/v3.mdx

```diff
@@ -594,7 +594,6 @@ Note that spaCy v3.0 now requires **Python 3.6+**.
 | `GoldParse`                                                                                  | [`Example`](/api/example)                                                                                                                                                                                                |
 | `GoldCorpus`                                                                                 | [`Corpus`](/api/corpus)                                                                                                                                                                                                  |
 | `KnowledgeBase.load_bulk`, `KnowledgeBase.dump`                                              | [`KnowledgeBase.from_disk`](/api/kb#from_disk), [`KnowledgeBase.to_disk`](/api/kb#to_disk)                                                                                                                               |
-| `KnowledgeBase.get_candidates`                                                               | [`KnowledgeBase.get_alias_candidates`](/api/kb#get_alias_candidates)                                                                                                                                                     |
```

Comment:
Do we need to remove this? This is really just the changelog page for v3.

Reply:
Reverted in https://github.com/explosion/spaCy/pull/12367/commits/5d1ecf1aa12629f96a803fc33a4a050b60ac9f61.